session/session_test.go

```diff
@@ -1650,6 +1650,10 @@ func (s *testSchemaSuite) TestTableReaderChunk(c *C) {
 	s.cluster.SplitTable(s.mvccStore, tbl.Meta().ID, 10)
 
 	tk.Se.GetSessionVars().DistSQLScanConcurrency = 1
+	tk.MustExec("set tidb_max_chunk_size = 2")
```

Comment:
`tidb_max_chunk_size` is the size of the batch the `Next()` function returned, the test should be pased with every kind of batch size.

Set it to 2 makes us be able to test the operators that return more than one batch.

Reply:
I mean this part:
```go
 	for {
 		err = rs.Next(context.TODO(), chk)
 		c.Assert(err, IsNil)
 		numRows := chk.NumRows()
 		if numRows == 0 {
 			break
 		}
 		for i := 0; i < numRows; i++ {
 			c.Assert(chk.GetRow(i).GetInt64(0), Equals, int64(count))
 			count++
 		}
 		numChunks++
 	}
 	c.Assert(count, Equals, 100)
 	c.Assert(numChunks, Equals, 50)
```
why we can assert `numChunks == 50` when not setting `tidb_max_chunk_size = 2`?

Reply:
Uh.. I think this is because the `tidb_max_chunk_size` session variable we created for test is set to 2 in this place: https://github.com/pingcap/tidb/blob/release-2.0/session/session.go#L1094